gno.land/pkg/integration/testing_node.go

```diff
@@ -59,6 +59,7 @@ func TestingNodeConfig(t TestingTS, gnoroot string, additionalTxs ...std.Tx) (*g
 
 	creator := crypto.MustAddressFromString(DefaultAccount_Address) // test1
 
+	params := LoadDefaultGenesisParamFile(t, gnoroot)
```

Comment:
@gfanton, I think there is a problem with Codecov. Can you check why this line isn't reported?

Reply:
Because it's only used by `txtar` files in `gno.land`, unfortunately, the `gno.land` `txtar` doesn't set up coverage yet. I suggest you ignore this for now. Ideally, I would like to take some time centralize the logic of `txtar` from both `gnovm` and `gno.land`.

Reply:
I see you want to avoid losing too much coverage by taking advantage of `txtar`. Let me see if I can find a shortcut.

Reply:
I'm just looking for my PR to be green, because I wrote tests.

Reply:
@moul 

So it looks like your issue has nothing to do with `txtar`.

By default, Go's test coverage tool only considers the packages that are directly imported by the package containing the tests. This means that if you have a package dependency chain like:  
-  **Package A** (contains tests) imports **Package B**  
-  **Package B** imports **Package C**

Coverage will be calculated for **Package A** and **Package B**, but **Package C** will not be included in the coverage report unless explicitly specified. This is because Go's default behavior is to only instrument the packages directly involved in the test run.

In your case:  
-  **Package A** is the package that holds the test file in `gno.land/cmd/gnoland`  
-  **Package B** is `gno.land/pkg/integration`, and it's the package that will actually run the `txtar`  
-  **Package C** is the package that contains `LoadDefaultGenesisParamFile` and will not be covered by your test  

There are 2 possible fixes for this:  
-  The first one, more global, is to tell Go to test and cover all packages under `gno.land` using `-coverpkg=github.com/gnolang/gno/gno.land/...`. This way, all packages and sub-package imports will be taken into consideration; it can be a good solution but at the expense of slower tests.  
-  The second solution, more specific to your PR, is to reduce the chain dependency of your test. That means simply moving your `params.txtar` from `gno.land/cmd/gnoland/testadata` to `gno.land/pkg/integration/testdata`.

Reply:
I'm in favor of option 1, unless "slower" means "way too much slower."

Reply:
Something like this: https://github.com/gnolang/gno/pull/3088?

Reply:
> Something like this: https://github.com/gnolang/gno/pull/3088?

see my comment https://github.com/gnolang/gno/pull/3088#issuecomment-2463450951

Reply:
Alternative option: https://github.com/gnolang/gno/pull/3090